src/legacy/ui/public/new_platform/new_platform.ts

```diff
@@ -17,29 +17,11 @@
  * under the License.
  */
 
-import { BasePathSetup } from '../../../../core/public/base_path';
-import { ChromeSetup } from '../../../../core/public/chrome';
-import { FatalErrorsSetup } from '../../../../core/public/fatal_errors';
-import { HttpSetup } from '../../../../core/public/http';
-import { I18nSetup } from '../../../../core/public/i18n';
-import { InjectedMetadataSetup } from '../../../../core/public/injected_metadata';
-import { NotificationsSetup } from '../../../../core/public/notifications';
-import { UiSettingsSetup } from '../../../../core/public/ui_settings';
-
-interface CoreSetup {
-  i18n: I18nSetup;
-  injectedMetadata: InjectedMetadataSetup;
-  fatalErrors: FatalErrorsSetup;
-  notifications: NotificationsSetup;
-  http: HttpSetup;
-  basePath: BasePathSetup;
-  uiSettings: UiSettingsSetup;
-  chrome: ChromeSetup;
-}
+import { CoreSetup } from '../../../../core/public/';
```

Comment:
You can remove this trailing slash.

```suggestion
import { CoreSetup } from '../../../../core/public';
```